src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const act of actions) {
+    switch (act.type) {
+      case 'addProperties':
+        Object.assign(state, act.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const key of act.keysToRemove) {
+          delete state[key];
+        }
+        break;
```

Comment:
Checklist: [CODE STYLE]: switch/case should always have default case for error handling.